### PR TITLE
fix: base image security

### PR DESCRIPTION
**Pull request by lijie0123**
Why:
- fix base image security
- 
img_v2_245f6c0e-6075-42b2-aa39-0ddcfa5f426g

- better makefile for parallel build

### CODE REVIEW

**Comment by wuhuizuo** [comment]
> Why:
> 
> * fix base image security
> * better makefile for parallel build

what are the security issues, any detail?

**Comment by lijie0123** [comment]
added scanned security issue

File: dockerfiles/bases/Makefile [hunk 1]
```diff
@@ -1,7 +1,12 @@
 Images  = $(shell gawk '/^\# hub/{printf "-t %s " , $$2}' $@/Dockerfile )
 
-all_bases := pingcap-base tidb-base tikv-base pd-base tools-base tiflash-base
-$(all_bases) :
+products_bases := tidb-base tikv-base pd-base tools-base tiflash-base
+pingcap-base :
 	docker buildx build $@ $(Images) --push --platform=linux/amd64,linux/arm64
 
-.PHONY : $(all_bases)
+$(products_bases) : pingcap-base
```
Comment (wuhuizuo) [inline dockerfiles/bases/Makefile:7]: Tiny: maybe we should use `skaffold` to make it more readable.

**Comment by wuhuizuo** [comment]
We should better split it with two pull request, one for `pingcap-base`, another for other product dockerfiles.

File: dockerfiles/bases/pingcap-base/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,8 @@
-# hub.pingcap.net/bases/pingcap-base:v1.6.0
-# hub.pingcap.net/bases/pingcap-base:v1.6
+# hub.pingcap.net/bases/pingcap-base:v1.7.0
+# hub.pingcap.net/bases/pingcap-base:v1.7
 # hub.pingcap.net/bases/pingcap-base:v1
 FROM rockylinux:9.2.20230513
 RUN --mount=from=busybox:1.36.1,source=/bin,target=/busybox/bin \
 	cp -a /busybox/bin/busybox /bin/busybox && \
+	dnf upgrade -y python3 less && \
```
Comment (wuhuizuo) [inline dockerfiles/bases/pingcap-base/Dockerfile:7]: Why update python3 and less package?

**Pull request merged**